domain/port/service/service.go

```diff
@@ -62,14 +63,6 @@ type Service struct {
 	st State
 }
 
-// NewService returns a new Service providing an API to manage the opened ports
-// for units.
-func NewService(st State) *Service {
-	return &Service{
-		st: st,
-	}
-}
```

Comment:
You'll want to keep this if you require migrations.

Reply:
Sure thing